src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -29,25 +37,80 @@ protected function configure()
      * @return void
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $sprykName = current((array)$input->getArgument(static::ARGUMENT_SPRYK));
+        if ($sprykName !== false) {
+            $this->dumpSprykOptions($output, $sprykName);
+
+            return;
+        }
+
+        $this->dumpAllSpryks($output);
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return void
+     */
+    protected function dumpAllSpryks(OutputInterface $output): void
     {
         $sprykDefinitions = $this->getFacade()->getSprykDefinitions();
-        $sprykDefinitions = $this->formatForTable($sprykDefinitions);
+        $sprykDefinitions = $this->formatSpryksTable($sprykDefinitions);
```

Comment:
I think when you rename it would be better to use `formatSpryks` as you do not format a table here.